README.md

```diff
@@ -795,10 +823,11 @@ _NOTE: Skip these steps if you are deploying without promoting changes._
      http https://service.explore.data.humancellatlas.org/health
      ```
 
-5. Look at the history of `CHANGELOG.yml` since the last release to determine
-   when reindexing is necessary. When in doubt assume yes. In that case run the
-   manual `reindex` job on the Gitlab pipeline representing the most recent
-   build on the current branch.
+6. In the case that you need to reindex run the manual `reindex` job on the 
+   Gitlab pipeline representing the most recent build on the current branch.
+
+7. Once reindexing is complete announce this in the #data-wranging Slack
```

Comment:
Might want to mention how to tell if reindexing is complete.

Reply:
Added